server/services/RunService.js

```diff
@@ -393,7 +410,27 @@ async function sequelizeRunsToJsonRuns(sequelizeRuns) {
             run_status_id: run.run_status_id,
             run_status: run.RunStatus.name,
             test_version_id: run.test_version_id,
-            testers: run.Users.map(u => u.id)
+            testers: run.Users.map(u => u.id),
+            tests: run.ApgExample.Tests.reduce((acc, test) => {
```

Comment:
getting the tests for a run is a little more complicated that this -- you need to use TestToAt to make sure that any test applies to a run with a specific AT. Not all test files for an APG pattern apply to all ATs.

For example, of all Checkbox APG example tests, 2 of them do not apply to VoiceOver, so those test need to be skipped in all the runs of VoiceOver.